Libraries/Animated/src/Animated.js

```diff
@@ -16,12 +16,22 @@ var AnimatedImplementation = require('AnimatedImplementation');
 var Image = require('Image');
 var Text = require('Text');
 var View = require('View');
-var ScrollView = require('ScrollView');
 
-module.exports = {
-  ...AnimatedImplementation,
+let AnimatedScrollView;
+
+const Animated = {
   View: AnimatedImplementation.createAnimatedComponent(View),
   Text: AnimatedImplementation.createAnimatedComponent(Text),
   Image: AnimatedImplementation.createAnimatedComponent(Image),
-  ScrollView: AnimatedImplementation.createAnimatedComponent(ScrollView),
+  get ScrollView() {
+    // Make this lazy to avoid circular reference.
+    if (!AnimatedScrollView) {
+      AnimatedScrollView = AnimatedImplementation.createAnimatedComponent(require('ScrollView'));
+    }
+    return AnimatedScrollView;
+  },
 };
+
+Object.assign((Animated: Object), AnimatedImplementation);
```

Comment:
Is there a way to express this properly in flow? I want to add props to an object using `Object.assign`. Can't create a new object here because it breaks the getter.